scripts/aws-cni-support.sh

```diff
@@ -23,7 +23,9 @@ LOG_DIR="/var/log/aws-routed-eni"
 # collecting L-IPAMD introspection data
 curl http://localhost:61678/v1/enis         > ${LOG_DIR}/eni.output
 curl http://localhost:61678/v1/pods         > ${LOG_DIR}/pod.output
-curl http://localhost:61678/v1/env-settings > ${LOG_DIR}/env.output
+curl http://localhost:61678/v1/networkutils-env-settings > ${LOG_DIR}/networkutils-env.output
+curl http://localhost:61678/v1/ipamd-env-settings > ${LOG_DIR}/ipamd-env.output
+curl http://localhost:61678/v1/eni-configs  > ${LOG_DIR}/eni-configs.out
```

Comment:
Nit: Why does the eni-configs log file have a `.out` extension when all the others have `.output`